consumer.py fails: AttributeError: 'Task' object has no attribute 'get_resources'

When pushing a smaple via the provided `push_sample.py` and trying to generate reports automatically by using the `consumer.py` the consumer fails with the following error message: 

```shell
[2020-09-29 11:06:34,593][INFO] Received new task - b92cdf0b-90ca-4737-ae60-649c9a79d0fb
[2020-09-29 11:06:34,594][INFO] Storing analysis 7f628b98-a79a-454d-96d2-0ca1dc519f6e into /tmp/drakrunnjs7itmi
[2020-09-29 11:06:34,595][ERROR] Failed to process task - b92cdf0b-90ca-4737-ae60-649c9a79d0fb
Traceback (most recent call last):
  File "/opt/venvs/drakrun/lib/python3.7/site-packages/karton2/karton.py", line 173, in internal_process
    self.process()
  File "consumer.py", line 27, in process
    for name, resource in self.current_task.get_resources():
AttributeError: 'Task' object has no attribute 'get_resources'
```

the created directory `/tmp/drakrunnjs7itmi` is empty afterwards.

Steps to reproduce the behavior:
1. Use provided `consumer.py` to generate reports automatically after pushing.
2. Error gets thrown on try to generation after analysis finished


I had to import the karton2 package via 
```python
import sys
sys.path.append('/opt/venvs/drakrun/lib/python3.7/site-packages/')
from karton2 import Karton, Config, Task, DirectoryResource
```
otherwise i only get an "package not found" error and I didn't find another way to get it running.
Importing it from `/opt/venvs/drakcore/lib/python3.7/site-packages/` did find the package too, but didn't work either for the generation.

Does not work if samples aree pushed via the Web-UI either.


`journalctl -e -u drakrun@2` just gets me this:

```shell
...
Sep 29 11:06:00 milano drakrun[1113]: WARNING:root:Failed to parse log entry:
Sep 29 11:06:00 milano drakrun[1113]: 'NoneType' object has no attribute 'replace'
Sep 29 11:06:31 milano drakrun[1113]: [2020-09-29 11:06:31,937][INFO] uploading artifacts
Sep 29 11:06:31 milano drakrun[1113]: [2020-09-29 11:06:31,938][DEBUG] Dispatched task d84c0df8-e757-443d-835f-b0ad7d00183a
Sep 29 11:06:34 milano drakrun[1113]: [2020-09-29 11:06:34,591][INFO] Task done - a85c4bed-25c7-4136-8b1a-51aa8958fdd5
```
probably because of the figs I mentioned in #216

--- a/examples/consumer.py
+++ b/examples/consumer.py
@@ -22,7 +22,7 @@ class DrakrunAnalysisConsumer(Karton):
 
         self.log.info(f"Storing analysis {analysis_uid} into {tmp_dir}")
 
-        for name, resource in self.current_task.get_resources():
+        for name, resource in self.current_task.iterate_resources():
             with open(os.path.join(tmp_dir, resource.name), 'wb') as f:
                 f.write(self.download_resource(resource).content)
 

